--- /osxcross/target/macports/pkgs/opt/local/lib/cmake/SDL2/sdl2-config.cmake.orig	2021-03-30 20:48:30.978665366 +0000
+++ /osxcross/target/macports/pkgs/opt/local/lib/cmake/SDL2/sdl2-config.cmake	2021-03-30 20:49:59.361202280 +0000
@@ -1,6 +1,6 @@
 # sdl2 cmake project-config input for ./configure scripts
 
-set(prefix "/opt/local") 
+set(prefix "/osxcross/target/macports/pkgs/opt/local") 
 set(exec_prefix "${prefix}")
 set(libdir "${exec_prefix}/lib")
 set(SDL2_PREFIX "/opt/local")
